src/client.js

```diff
@@ -28,20 +28,12 @@ class NgrokClient {
       }
     } catch (error) {
       let clientError;
```

Comment:
Couple of things here:

1. Now that we're not doing a `try ... catch` we don't need to create the variable at the start.
2. I think it is still worth catching errors from JSON.parse should the response body be incorrect JSON
3. I think the double nested ternary statement makes it hard to follow the logic 
4. We repeat some logic that we can put in a helper function

Could we do something like this:

```js
// .. defined above

function getResponse(error) {
  if (error.response && error.response.body) {
    let response;
    try {
      response = JSON.parse(error.response.body);
    } catch (e) {
     throw new NgrokClientError(e.msg, e.details);
    }
    return response;
  }

  return error;
}

// ... then instead of 

const response = getResponse(error);
throw new NgrokClientError(error.msg, error.details, response);

// ... and

} catch (e) {
  const response = getResponse(e);
  throw new NgrokClientError(e.msg, e.response, response);
}


```